server/src/instant/db/model/triple.clj

```diff
@@ -19,8 +19,8 @@
 ;; In the future, we may want to _retract_ the triple if the value is nil
 (defn value? [x]
   (or (string? x) (uuid? x) (number? x) (nil? x) (boolean? x)
-      (sequential? x) (associative? x)))
-
+      (sequential? x) (associative? x)
+      (instance? java.time.Instant x)))
```

Comment:
I am not 100% sure if this is a good idea. Alternatively we could have a separate spec, which we use in datalog for the constant portion of the queries